src/test/java/io/openliberty/tools/intellij/it/fixtures/ProjectFrameFixture.java

```diff
@@ -145,6 +146,34 @@ public ComponentFixture getContentComboLabel(String... xpathVars) {
                 Duration.ofSeconds(Integer.parseInt(waitTime)));
     }
 
+    /**
+     * Returns the ComponentFixture object associated with the ProjectViewTree class.
+     *
+     * @param xpathVars The Locator custom variables: text
+     * @return The ComponentFixture object associated with the ProjectViewTree class.
+     */
+    public ComponentFixture getProjectViewTree(String... xpathVars) {
+        String visibleText = xpathVars[0];
+
+        return find(ComponentFixture.class,
+                byXpath("//div[@class='ProjectViewTree' and contains(@visible_text, '" + visibleText + "')]"),
+                Duration.ofMinutes(1));
+    }
+
+    /**
+     * Returns the ComponentFixture object associated with the ProjectViewTree class.
+     *
+     * @param xpathVars The Locator custom variables: text, waitTime(seconds)
+     * @return The ComponentFixture object associated with the ProjectViewTree class.
+     */
+    public ComponentFixture getProjectViewTree2(String... xpathVars) {
```

Comment:
Could we improve the names of `getProjectViewTree` and `getProjectViewTree2`? - I see one method is looking for visibleText and the other has a waitTime